src/convertToObject.js

```diff
@@ -6,7 +6,27 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const result = {};
+  const splited = sourceString.split(';');
+
+  for (const item of splited) {
```

Comment:
Avoid using loops for this task. Instead, consider using iteration methods like 'forEach' to process the array.